app/src/main/java/io/intrepid/contest/models/ScoreWeight.java

```diff
@@ -0,0 +1,25 @@
+package io.intrepid.contest.models;
+
+import android.support.annotation.StringRes;
+
+public class ScoreWeight {
+
+    private int weightValue;
+    private
+    @StringRes
+    int weightName;
```

Comment:
I think the @StringRes shifts the line down when I do an autofix. I'll fix this.